sys/linux/dev_bifrost.txt

```diff
@@ -0,0 +1,370 @@
+# Copyright 2020 syzkaller project authors. All rights reserved.
```

Comment:
I think this file also needs to be added here:
https://github.com/google/syzkaller/blob/master/sys/syz-extract/extract.go#L262-L267
Since these headers are not upstream, `make extract` on upstream tree will start failing otherwise.

Reply:
Done.